tests/Kernel/fixtures/rendering.yml

```diff
@@ -736,6 +736,30 @@
       'div.ecl-content-item-block__item.ecl-col-md-2:nth-child(4) a.ecl-link': 'A simple link 4'
       'div.ecl-content-item-block__item.ecl-col-md-2:nth-child(5) a.ecl-link': 'A simple link 5'
       'div.ecl-content-item-block__item.ecl-col-md-2:nth-child(6) a.ecl-link': 'A simple link 6'
+- array:
```

Comment:
It would be nice to have test coverage for cases when we have empty:
- links
- summary
- that summary should be visible when we have empty links